### PR TITLE
Option to disable the validator creation

### LINKED ISSUE #4: Option to disable the validator creation
So far, we've hardcoded to disable the validator creation. It caused for us to manage two different versions: `v0.0.0` and `v0.0.0-internal`.
Let's have an option to disable the validator creation. Then, we can keep only one version scheme: `v0.0.0`.

The details would be designed soon.

### PR DESCRIPTION
**Pull request by youngjoon-lee**
Closes #4

### CODE REVIEW

File: x/staking/handler.go [hunk 1]
```diff
@@ -105,6 +105,10 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) []abci.ValidatorUpdate {
 // now we just perform action and save
 
 func handleMsgCreateValidator(ctx sdk.Context, msg types.MsgCreateValidator, k keeper.Keeper) sdk.Result {
+	if ctx.ValidatorCreationDisallowed() {
+		return types.ErrNotAllowed(DefaultCodespace).Result()
+	}
```
Comment (youngjoon-lee) [inline x/staking/handler.go:110]: Allow the validator creation when the height is 0 (for genesis).

Comment (youngjoon-lee) [reply x/staking/handler.go]: Done 
f0a545b 

@cl9200

**Review by cl9200 — APPROVED** [review]
:+1:

**Comment by youngjoon-lee** [comment]
Released as [v0.37.15](https://github.com/medibloc/cosmos-sdk/releases/tag/v0.35.7-internal) (No `*-internal` anymore).

**Pull request merged**